<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AyarlarDialog</class>
 <widget class="QDialog" name="AyarlarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ayarlar</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tabGenel">
    <attribute name="title">
     <string>Genel</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabKullanicilar">
    <attribute name="title">
     <string>Kullanıcılar</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kullanıcılar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="YeniPushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Yeni</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DuzenlePushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Düzenle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SilPushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sil</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>341</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Kullanıcı Yetkileri</string>
     </property>
     <widget class="QCheckBox" name="KasacheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Kasa giriş/çıkış yapabilir</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="IadecheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>İade alabilir</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="StokcheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stok işlemleri yapabilir</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="KullanicilarlistWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Yazdırma</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fiş yazıcısı</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fisYazicisicomboBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="herZamancheckBox">
        <property name="text">
         <string>Her zaman fiş yazdır</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabVeritabani">
    <attribute name="title">
     <string>Veritabanı</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Şirket Bilgileri</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Adı</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Vergi Dairesi</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Vergi No</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>İl</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>İlçe</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Adres</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Telefon</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
